includes/free/form-element.php

```diff
@@ -398,11 +424,17 @@ public static function render_registration_settings() {
             <td>
                 <select disabled name="">
                     <?php
-                    $pages = get_posts(  [ 'numberposts' => -1, 'post_type' => 'page'] );
+                    $pages = get_posts(
+                        [
+							'numberposts' => -1,
+							'post_type' => 'page',
+						]
+                    );
```

Comment:
The formatting is not correct, need to fix.

Reply:
I used `composer run phpcbf`